docs/content/en/docs/tasks/packages/prereq.md

```diff
@@ -0,0 +1,33 @@
+## Prerequisites
+Before installing any curated packages for EKS Anywhere, do the following:
+
+* Check that the version of `eksctl anywhere` is `v0.9.0` or above with the following commands:
+    ```bash
+    eksctl anywhere version
```

Comment:
The below output isn't checking the version. It's for the package controller output

Reply:
That was my mistake. I'll fix that.